Add stack for Prophecy

vladsaveliev opened the pull request.

illusional commented:
Hey @vladsaveliev, you'll need to add the `prophecy` to the `tokens/repository-map.json` file to ensure permissions are generated properly. Is this a seqr project, if so will need to added to `depends_on`.

illusional commented:
And also a plug for https://github.com/populationgenomics/analysis-runner/pull/300 for making it easier to create datasets.

vladsaveliev commented:
Right, added it into `tokens/repository-map.json` (and also added the missing `bioheart`, and added missing repos for `nagim`)

Those are not going to be on seqr as far as I understand, but if anything changes, will add the dependencies later
